src/display/editor/tools.js

```diff
@@ -1432,6 +1463,13 @@ class AnnotationEditorUIManager {
     return this.#selectedEditors.size !== 0;
   }
 
+  get isEnterHandled() {
+    return (
+      this.#selectedEditors.size === 1 &&
+      [...this.#selectedEditors][0].isEnterHandled
```

Comment:
We've now using two different ways of accessing the first element of this `Set`, note [this existing code](https://github.com/mozilla/pdf.js/blob/552cc059cc40fb3e5e7d6185c85a567821b6416c/src/display/editor/tools.js#L1777). How about introducing a common helper for that, e.g. something like the following?
```javascript
get firstSelectedEditor() {
  return this.#selectedEditors.keys().next().value;
}